misc/dbt-materialize/dbt/include/materialize/macros/deploy/deploy_init.sql

```diff
@@ -43,6 +43,8 @@
     {% endif %}
 {% endfor %}
 
+{{ deploy_permission_validation(clusters) }}
```

Comment:
I know this isn't meant to be called directly by users but probably good to nonetheless stick with the `deploy_<VERB>` naming scheme, so `deploy_validate_permissions`.